src/SimpleSAML/Metadata/MetaDataStorageHandlerSerialize.php

```diff
@@ -271,8 +271,9 @@ public function deleteMetadata(string $entityId, string $set): void
     /**
      * This function loads the metadata for entity IDs in $entityIds. It is returned as an associative array
      * where the key is the entity id. An empty array may be returned if no matching entities were found
-     * @param array $entityIds The entity ids to load
-     * @param string $set The set we want to get metadata from.
+     *
+     * @param  array<string, int> $entityIds The entity ids to load
```

Comment:
I'm actually quite opposed to changing all these method signatures for any SSP 2.X release.  They are part of the public API contract for people extending SSP by creating their own metadata sources and this is not a backward compatible change.  My vote is to keep `$entityIds` as string[] arrays to make everyone's lives easier when they upgrade to 2.4.